mantidproject/mantidimaging
# Now asks the user to delete data or ignore it

Should be merged after #773 

Shows a prompt to the user on startup if it finds any shared arrays present.

This is only a temporary "band-aid" for this release, to avoid having to merge #770 due to the seen memory leak issues. That's why it's not part of the user welcome screen, and also because I couldn't figure out a better way of grabbing the user's attention

To test:

- Start up a mantid imaging
- Load data in it
- Start up another one
- Select Ignore on the prompt
- Load data in it
- The first one should still work OK, try to applying an operation

Fixes #774
